Sources/PackageDescription/Version.swift

```diff
@@ -12,31 +12,31 @@
 
 /// A version according to the semantic versioning specification.
 ///
-/// A package version is a three period-separated integer, for example `1.0.0`. It must conform to the semantic versioning standard in order to ensure
+/// A package version consists of three integers separated by periods, for example `1.0.0`. It must conform to the semantic versioning standard in order to ensure
 /// that your package behaves in a predictable manner once developers update their
 /// package dependency to a newer version. To achieve predictability, the semantic versioning specification proposes a set of rules and
 /// requirements that dictate how version numbers are assigned and incremented. To learn more about the semantic versioning specification, visit
-/// [semver.org](www.semver.org).
+/// [Semantic Versioning 2.0.0](www.semver.org).
 ///
 /// **The Major Version**
```

Comment:
unintentional leftover heading?

Reply:
Probably... thanks for pointing this out!